src/Microsoft.Extensions.Configuration.UserSecrets/ConfigurationExtensions.cs

```diff
@@ -3,15 +3,19 @@
 
 using System;
 using System.IO;
+using System.Reflection;
 using Microsoft.Extensions.Configuration.UserSecrets;
+using Microsoft.Extensions.Configuration.UserSecrets.Internal;
 using Microsoft.Extensions.FileProviders;
 
 namespace Microsoft.Extensions.Configuration
 {
+    /// <summary>
+    /// Configuration extensions for adding user secrets configuration source.
+    /// </summary>
     public static class ConfigurationExtensions
     {
-        private const string Secrets_File_Name = "secrets.json";
-
+#if NET451 || NETSTANDARD1_5
```

Comment:
Changing the basic `public static IConfigurationBuilder AddUserSecrets(this IConfigurationBuilder configuration)` to now throw on netstandard1.3 is a breaking change and not allowed. I.e. I should be able to upgrade my `Microsoft.Extensions.Configuration.UserSecret` reference after this change in a working app and have it function _exactly_ the same as it was before